lib/b3fs/src/hasher/collector.rs

```diff
@@ -2,12 +2,14 @@ use std::array::TryFromSliceError;
 
 use arrayref::array_ref;
 use thiserror::Error;
-use tokio::io;
 
 use super::b3::KEY_LEN;
 use super::byte_hasher::{Blake3Hasher, BlockHasher};
 use super::HashTreeCollector;
+#[cfg(feature = "async")]
 use crate::bucket::errors;
+#[cfg(not(feature = "async"))]
```

Comment:
I think it is not needed to `not` Feature flag

Reply:
The `sync` module only exists when the `async` feature is disabled, so the import `use crate::sync::bucket::errors;` will fail whenever it's enabled. and it's enabled by default.

Reply:
Why we are not leaving sync module as a part of the crate and only enable async on async context? I think if we do that, we are not going to need to put `not(feature = "async")` in a bunch of places.

Reply:
We can do that, but then we will bring in the `sync` module every time. So when we bring in `b3fs` in the `blockstore` and the `sdk` (and wherever else we use it), we will also bring in the `sync` module, even though we don't need it.